src/Controls/tests/Core.UnitTests/ControlsMapperTests.cs

```diff
@@ -14,13 +14,12 @@
 
 namespace Microsoft.Maui.Controls.Core.UnitTests
 {
-	[Collection("ControlsMapperTests")]
+	[Collection("TestsAccessingStaticHandlers")]
 	public class ControlsMapperTests : BaseTestFixture
 	{
 		MauiContext SetupMauiContext()
 		{
 			var mauiApp1 = MauiApp.CreateBuilder()
-				.UseMauiApp<ApplicationStub>()
```

Comment:
ApplicationSTub isn't really needed for this set of tests since we're just validating that the basic replace method work on a handler that's local to this test